infra/dev/start-env.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# start only the container for hosting the file
+USER_ID=$(id -u) GROUP_ID=$(id -g) docker-compose up laravel nginx db adminer
```

Comment:
Same as for `setup-env.sh` and why are you specifying the containers to up ? Isn't `$ docker-compose up` enough?

Reply:
no we need the information giving in the args user_id and group_id